src/extensions.rs

```diff
@@ -195,4 +195,15 @@ impl<T: AsRef<str>> T {
         let index = s.find(c)?;
         Some((&s[..index], &s[index + c.len_utf8()..]))
     }
+
+    fn split_at_word<'a>(&'a self, split_at: &str) -> (String, String) {
```

Comment:
Could you add a short doc-comment here, explaining what the function does? specifically, providing one example of how it'd behaves  (use triple `///` for doc-comments)
i.e.:
```
Splits the string into two halves, separated by a given word.

## Example
`"foo bar baz".split_at_word("bar")  // ---> ("foo", "baz")`
```

I think in this case that's currently actually wrong, as currently the left side will be `foo bar`, right? That should prolly be fixed, see the next review (if i'm wrong plox ignore that one, i might be stupid rn, try it)